documentation/docs/globals/fetch.mdx

```diff
@@ -56,6 +56,9 @@ fetch(resource, options)
       - *Fastly-specific*
     - `cacheOverride` _**Fastly-specific**_
     - `cacheKey` _**Fastly-specific**_
+    - `fastly` _**Fastly-specific**_
+      - `decompressGzip`_: boolean_ _**optional**_
```

Comment:
Should we also add this in our `types` folder as well? Or did you want to keep it "experimental"? We can still type it even as experimental I think?

Reply:
Yes definitely, I have added that in [`df063dd` (#497)](https://github.com/fastly/js-compute-runtime/pull/497/commits/df063ddc249d9116ee49860165dc7801a5bcf6ca)